app/controllers/application_controller.rb

```diff
@@ -2,4 +2,40 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  helper_method :current_robot
+  before_action :current_order
```

Comment:
The methods themselves should be private, not the listing of the `before_action`. Because ONLY this controller _should_ and _will_ be accessing these methods.